doc/classes/ReferenceRect.xml

```diff
@@ -4,14 +4,15 @@
 		Reference frame for GUI.
 	</brief_description>
 	<description>
-		Reference frame for GUI. It's just like an empty control, except a red box is displayed while editing around its size at all times.
+		Reference frame for GUI. It's just like an empty control, except a [member border_color] is displayed while editing around its size at all times.
```

Comment:
I agree with the change need, although it is unclear what is displayed here: to keep the information that a box is displayed, I'd go with something like `except an outline/box colored with [member border_color] is displayed`.
What is displayed exactly btw? It is an outline (hollow) or a box (full)?

Reply:
It's just an outline border. I believe this was intended to emphasize the currently active control